bundles/pcmjava/tools.vitruv.applications.pcmjava.gplimplementation.pojotransformations.java2pcm/src/tools/vitruv/applications/pcmjava/gplimplementation/pojotransformations/java2pcm/Java2PcmChangeProcessor.xtend

```diff
@@ -20,16 +20,16 @@ class Java2PcmChangeProcessor extends TransformationExecutorChangeProcessor {
 	new() {
 		super(new JavaDomainProvider().domain, new PcmDomainProvider().domain)
 
```

Comment:
This is wrong and makes the GPL implementation tests not work (as there are not executed transformations anymore).
The Reactions implementation should (in its final version) not depend on the GPL implementation, so that the GPL implementation can stay as it was before.